modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/catalog/resource/v1_0/test/SkuResourceTest.java

```diff
@@ -150,6 +216,24 @@ protected Sku testGraphQLSku_addSku() throws Exception {
 		return _addCPInstance(randomSku());
 	}
 
+	@Override
+	protected Sku
+			testPostChannelByExternalReferenceCodeProductProductExternalReferenceCodeSku_addSku(
+				Sku sku)
+		throws Exception {
+
+		return testGetChannelProductSku_addSku();
+	}
+
+	@Override
+	protected Sku
+			testPostChannelByExternalReferenceCodeProductProductExternalReferenceCodeSkuBySkuOption_addSku(
+				Sku sku)
+		throws Exception {
+
+		return testGetChannelProductSku_addSku();
```

Comment:
same

Reply:
changed to return _addCPInstance(randomSku());